src/components/EditCoursePage/index.jsx

```diff
@@ -85,7 +85,7 @@ class EditCoursePage extends React.Component {
   }
 
   render() {
-    if (!this.props.courseInfo || !this.props.courseOptions) {
+    if (!this.props.courseInfo || !this.props.courseOptions || !this.props.courseRunOptions) {
       return (
         <StatusAlert
           id="error"
```

Comment:
We should add an action the user can take.

For example, `Please try to reload the page and if this does not work contact support`.

Reply:
Happy to add that in, but if this scenario happens, I think we are in a bigger mess. This error messages when the entire object does not exist i.e. none of the `{data: {}, isFetching: False, error: null}`

Reply:
The main thing is contact support.  In the future, we should add a direct link to support of some fashion.  Even just an email.